### ISSUE TITLE
Need config option for "Stop all non-scrap objects from showing value when they do not actually have any"

### ISSUE THREAD

**Issue by K0nomi** [Shaosil/LethalCompanyMods-GeneralImprovements#232]
This fix causes some jank with the LethalCasino (at the very least), and there doesn't seem to be a way to specifically disable it.
This option causes keys to show as 0 rather than 3, to reflect the amount they actually sell for, but this causes issues with things that actually use this value. 
In LethalCasino, you are able to gamble a key up to a high value, then convert it's value to chips which do sell for money. This gives meaning to the key's value, so having it display as 0 is a detriment. It could be argued that this case is an inconsistency with LethalCasino, but the issue could affect other items/mods, and some users may not want the change due to it messing with things.
Basically, this needs a config option for compatibility purposes
